eskip/eskip.go

```diff
@@ -446,17 +444,25 @@ func (t BackendType) String() string {
 }
 
 // Expects exactly n arguments of type string, or fails.
-func getStringArgs(n int, args []interface{}) ([]string, error) {
-	if len(args) != n {
-		return nil, invalidPredicateArgCountError
+func getStringArgs(p *Predicate, n int) ([]string, error) {
```

Comment:
Is it worth changing the param order?

Reply:
I think its a matter of taste, I somehow like `getStringArgs(p, 2)` more than  `getStringArgs(2, p)` at the call site

Reply:
Its a helper, we could also make it a member although very specific to built-in predicates. Then the call would be `p.getStringArgs(2)`